extensions/vscode/src/activation/activate.ts

```diff
@@ -116,9 +116,7 @@ export async function activateExtension(context: vscode.ExtensionContext) {
 // Custom Layout settings that we want default for PearAPP
 const setupPearAPPLayout = async (context: vscode.ExtensionContext) => {
   // move pearai extension to auxiliary bar (secondary side bar) if there is a folder open
-  if(vscode.workspace.workspaceFolders){
-    vscode.commands.executeCommand("workbench.action.movePearExtensionToAuxBar");
-  }
+  vscode.commands.executeCommand("workbench.action.movePearExtensionToAuxBar");
```

Comment:
hmm the if condition was original put to prevent the opening of the sidebar on the "new window" screen, where it would open for like half a second and close. if we remove it here, we'd still need a way to prevent that.

Reply:
ill try to do this from inside of vscode, 
extension takes time to activate

Reply:
i tried to do this from inside vscode, couldn't figure it out, so pushed a change in this pr to hide sidebars on new window if no workspace is open.